src/main/java/com/GG/T9AgeCombat/models/Unit.java

```diff
@@ -48,7 +50,15 @@
     @NonFinal
     Integer reRollToHitLessThan;
     @NonFinal
-    Boolean hasReRollToWound;
+    Integer reRollToHitGreaterThan;
+    @NonFinal
+    Integer reRollToWoundLessThan;
+    @NonFinal
+    Integer reRollToWoundGreaterThan;
+    @NonFinal
+    Integer reRollArmorSaveLessThan;
+    @NonFinal
+    Integer reRollArmorSaveGreaterThan;
```

Comment:
Why don't we initialize these to their defaults, then our getters simple return the current value.

Reply:
From the DB call? Or is there a way to set the default other than remembering to add it into each builder?